model/src/mpicomm.F90

```diff
@@ -0,0 +1,67 @@
+!> @file
+!> @brief Defines MPI communicator id as constants for global use.
+!>
+!> @author H. L. Tolman  @date 05-Jun-2018
+!>
+#include "w3macros.h"
+
+!>
+!> @brief Define some mpi constants for global use
+!>
+!> @author  J. M. Sexton  @date 19-Jul-2024
+!>
+!
+#ifndef ENDIANNESS
```

Comment:
Can you help me understand why this is added here?

Reply:
This file was modelled off of constants, this ifdef is probably not needed.